docs/madr/decisions/070-resource-identifier.md

```diff
@@ -268,18 +268,23 @@ resource-identifier = *(ALPHA / DIGIT / "-" / "." / "_" / "~" )
 
 As a delimiter we can use only characters that can't be present in resource identifier components – `name`, `mesh`, `namespace`, `zone`, `resourceType` and `sectionName`:
 
-* `name` and `zone`
+* `name`
   * contain no more than 253 characters
   * contain only lowercase alphanumeric characters, '-' or '.'
```

Comment:
@lobkovilya What does the `name` mean here? Is it a Dataplane resource name or another resource name?

Reply:
every resource has `name`, so it's not just Dataplane's name

Reply:
Let me think about it twice.